specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/LinkedService.json

```diff
@@ -4229,6 +4229,18 @@
             "description": "Type: string (or Expression with resultType string)."
           }
         },
+        "newClusterDriverNodeType": {
+          "type": "object",
```

Comment:
type says object, but description says string ?

Reply:
arent expressions string ?

Reply:
The expressions are modeled with "Expression" type, and since Autorest is still on OpenApi 2.0, it doesn't support oneOf, hence we use object types everywhere we want to support expressions.

Reply:
new writable fields to existing api-version? Shouldnt these go in newer api-version?

Reply:
These are not required properties - why would they go into new API version? We have been adding new properties to our entities all the time and didn't need to update the API version, as long as they are not required.